empress/reconcile/recon_vis/utils.py

```diff
@@ -9,22 +9,29 @@
 from empress.reconcile.recon_vis import tree
 from empress.reconcile.recon_vis.tree import NodeLayout
 from empress.reconcile.recon_vis.tree import TreeType
+from enum import Enum
 
 __all__ = ['dict_to_tree', 'dict_to_reconciliation', 'build_trees_with_temporal_order']
 
+class ConsistencyType(Enum):
+    """ Defines type of the temporal consistency of a reconciliation """
+    STRONG_CONSISTENCY = 1
+    WEAK_CONSISTENCY = 2
+    NO_CONSISTENCY = 3
+
 # Master utility function coverts from dictionaries to objects
 
 def convert_to_objects(host_dict, parasite_dict, recon_dict):
     """
     :param host_dict - dictionary representation of host tree
     :param parasite_dict - dictionary representation of parasite tree
     :param recon_dict - dictionary representation of reconciliation
-    :return - corresondoing host_tree and parasite_tree Tree objects
-        and recon Reconciliation object
+    :return - corresondoing host_tree and parasite_tree Tree objects,
+        recon Reconciliation object and a ConsistencyType
     """
-    host_tree, parasite_tree, ok = build_trees_with_temporal_order(host_dict, parasite_dict, recon_dict)
+    host_tree, parasite_tree, consistency_type = build_trees_with_temporal_order(host_dict, parasite_dict, recon_dict)
```

Comment:
I think this function should be removed. We don't want to modify this function every time we modify `build_trees_with_temporal_order`. I think the user could run the two lines themselves. Would you mind looking for the occurrences of `convert_to_objects` and change it to `build_trees_with_temporal_order` and `dict_to_reconciliation`?